README_android.txt

```diff
@@ -47,18 +37,26 @@ Make NDK standalone toolchain
 Next you need to setup a standalone NDK toolchain. Set an environment
 variable to point to the desired location of the Android toolchain:
 
-    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-x86_64
```

Comment:
I think arm should be the default in instructions. Building APKs for x86(_64) is a niche case.

Reply:
I think I made a note somewhere of all the different architectures you can build for and everything is exactly the same for each. But yes, I can change the default. I was just thinking that people might not have an Android device and then are surprised how slow their game is in the emulator, while if they build with x86_64 the emulator will run at native speed.